src/hotspot/share/jvmci/jvmci_globals.hpp

```diff
@@ -157,6 +157,11 @@ class fileStream;
           "error data to this file"                                         \
           "[default: ./" LIBJVMCI_ERR_FILE "] (%p replaced with pid)")      \
                                                                             \
+  product(bool, LibJVMCICompilerThreadHidden, true, DIAGNOSTIC,             \
+          "If true then native JVMCI compiler threads are hidden from "     \
+          "JVMTI and FlightRecorder.  This must be set to false if you"     \
```

Comment:
`you"` -> `you "`
`threads"` -> `threads."`

As far as I can help, this help text is never printed (is it?) but it may as well be properly formatted.

Reply:
Fixed.  It's not in the product but it does function a documentation for the options.